usecases/decision_usecase.go

```diff
@@ -103,6 +108,22 @@ func (usecase *DecisionUsecase) GetDecision(ctx context.Context, decisionId stri
 	return decision, nil
 }
 
+func (usecase *DecisionUsecase) GetDecisionsByOutcomeAndScore(ctx context.Context,
+	testrunId string,
+) ([]models.DecisionsByVersionByOutcome, error) {
+	testrun, errTestRun := usecase.scenarioTestRunRepository.GetTestRunByID(ctx,
+		usecase.executorFactory.NewExecutor(), testrunId)
+	if errTestRun != nil {
+		return nil, errTestRun
+	}
+	decisions, err := usecase.repository.DecisionsByOutcomeAndScore(ctx,
```

Comment:
reading this, there is one last thing we have forgotten I think (or perhaps it's more a remark that concerns https://github.com/checkmarble/marble-backend/pull/756): the test run may have stopped before its initially set expiry date.
So two options:
- either we update testrun.Expires when we effectively stop a test run (change to do in the other PR)
- or we use a distinct field to keep a trace of the initially planned stop date and the actual stop date
(I think option 1 is fine, and in this case we don't need to change anything here)

Reply:
noted.. let's put this one on the testrun, on another pr..